client/state/activity-log/actions.js

```diff
@@ -310,27 +308,31 @@ export function updateRewindBackupProgress( siteId, downloadId, progress ) {
 /**
  * Update the status of the backup creation when it errors.
  *
- * @param  {string|number} siteId The site ID
- * @param  {string}        error  Error code
- * @return {object}               Action object
+ * @param  {string|number} siteId     The site ID
+ * @param  {number}        downloadId Id of the backup being created.
+ * @param  {string}        error      Info about downloadable backup and error.
+ * @return {object}                   Action object
  */
-export function rewindBackupUpdateError( siteId, error ) {
+export function rewindBackupUpdateError( siteId, downloadId, error ) {
 	return {
 		type: REWIND_BACKUP_UPDATE_ERROR,
 		siteId,
-		error,
+		downloadId,
+		...error,
```

Comment:
According to the JS docblock this is a string, but we're using the object spread operator here.

Reply:
My bad on the jsdoc